chia/full_node/fee_tracker.py

```diff
@@ -62,12 +62,23 @@ def get_estimate_time_intervals() -> List[uint64]:
 
 
 def get_bucket_index(sorted_buckets: SortedDict, fee_rate: float) -> int:
+    if len(sorted_buckets) < 1:
+        raise RuntimeError("get_bucket_index: sorted_buckets is invalid")
+
     if fee_rate in sorted_buckets:
         bucket_index = sorted_buckets[fee_rate]
```

Comment:
I still think you should remove this case, and always exercise the `bisec_left()` case. Is there a good reason to keep this? I fear it may obscure issues.

Reply:
I took your suggestion here. I believe the code is okay in both cases. All the `equals` case gives us is the property that the bucket upper bound is inclusive. One float epsilon won't make a difference in such large buckets, for sure!

As you point out, it is risky in general to use `float equals`, but this is not one of those cases - failure of an expected `float equals here` will always run the bisect case.

Reply:
I agree with that. I don't think the risk is that the case where the float equals the boundary is a bug, I think having two separate ways of doing the same thing adds extra test burden with no upside (afaict).